config/go-app-ussd-registration.eng_ZA.json

```diff
@@ -721,4 +721,4 @@
         null,
         ""
     ]
-}
+}
```

Comment:
nitpick: newline

Reply:
Checking that. I'm not sure why Github is complaining there.